icu4c/source/i18n/timezone.cpp

```diff
@@ -527,6 +527,11 @@ TimeZone::detectHostTimeZone()
 
 // -------------------------------------
 
+static UMutex *gMutex() {
+    static UMutex* m = new UMutex();
+    return m;
```

Comment:
The common idiom for ICU mutex creation just changed again, to avoid both memory leaks and order-of-destruction problems. It now should look like

      static UMutex *m = STATIC_NEW(UMutex);
      return m;

Reply:
done